web/src/message_lists.js

```diff
@@ -18,6 +20,10 @@ export function all_rendered_message_lists() {
     return rendered_message_lists;
 }
 
+export function all_current_message_rows() {
+    return $(`#${current.table_name}.message_table .message_row`);
```

Comment:
This implicitly assumes `current` is always defined; that is safe at present, but won't be in the future when we stop having it be the case that `message_lists.current` is set when viewing recent conversations.